docs/pages/setup/admin/trustedclusters.mdx

```diff
@@ -160,19 +204,70 @@ $ tctl tokens ls
 # ... or delete/revoke an invitation:
 $ tctl tokens rm (=presets.tokens.first=)
 ```
+The token created above can be used multiple times and has
+an expiration time of 5 minutes.
 
-Users of Teleport will recognize that this is the same way you would add any
-node to a cluster.  The token created above can be used multiple times and has
+<Admonition title="Security implications" type="warning">
+
+Consider the security implications when deciding which token method to use.
+Short-lived tokens decrease the window for an attack but will require any
+automation which uses these tokens to refresh them regularly.
+
+</Admonition>
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Cloud">
+
+You can create a join token on the fly using the `tctl` tool.
+
+<Admonition
+  type="tip"
+  title="Important"
+>
+
+  It's important to note that join tokens are only used to establish a
+  connection for the first time. Clusters will exchange certificates and
+  won't use tokens to re-establish their connection afterward.
+
+</Admonition>
+
+To create a token using the CLI tool, execute these commands on your development
+machine:
+
+```code
+# Generates a Trusted Cluster token to allow an inbound connection from a leaf cluster:
+$ tctl tokens add --type=trusted_cluster --ttl=5m
+# Example output:
+# The cluster invite token: (=presets.tokens.first=)
+# This token will expire in 5 minutes
+
+# Generates a Trusted Cluster token with labels.
+# Every cluster joined using this token will inherit env:prod labels.
+$ tctl tokens add --type=trusted_cluster --labels=env=prod
+
+# You can also list the outstanding non-expired tokens:
+$ tctl tokens ls
+
+# ... or delete/revoke an invitation:
+$ tctl tokens rm (=presets.tokens.first=)
+```
+The token created above can be used multiple times and has
 an expiration time of 5 minutes.
 
+</TabItem>
+</Tabs>
+
+
+Users of Teleport will recognize that this is the same way you would add any
+Node to a cluster. 
+
 Now, the administrator of "leaf" must create the following resource file:
```

Comment:
of **the** leaf

Reply:
I think skipping the article is okay here since "leaf" is the name of the cluster, but it does sound a bit odd. I've suggested using "the leaf cluster" instead.